### PR TITLE
Add logging details to transcripts processing

### PR DESCRIPTION
**Pull request by albandum**
## Description

I was young and innocent and put pretty much useless logging without details about files being transcribed

### CODE REVIEW

File: front/temporal/labs/activities.ts [hunk 1]
```diff
@@ -6,6 +6,7 @@ import {
   minTranscriptsSize,
 } from "@dust-tt/types";
 import { Err } from "@dust-tt/types";
+import { cons } from "fp-ts/lib/ReadonlyNonEmptyArray";
```
Comment (flvndvd) [inline front/temporal/labs/activities.ts:9]: 🤔

File: front/temporal/labs/activities.ts [hunk 2]
```diff
@@ -146,14 +147,21 @@ export async function processTranscriptActivity(
   }
 
   localLogger.info(
```
Comment (flvndvd) [inline front/temporal/labs/activities.ts:149]: `fileId` is already passed to the `localLogger` at creation. You might want to add the userId in the `localLogger` though.

**Review by flvndvd — COMMENTED** [review]
You can leverage more the localLogger to avoid passing everything, everytime 😉

File: front/temporal/labs/activities.ts (hunk 2 above)
Comment (albandum) [reply front/temporal/labs/activities.ts:149]: Noice. Changed.

**Review by flvndvd — APPROVED** [review]

**Pull request merged**